README.md

```diff
@@ -78,26 +78,26 @@ def print_array(array)
 end
 ```
 
-Snippet 7 - Big O:
+Snippet 7 - Big O: O(log n)
 ```ruby
 # this is insertion sort
-(2..num.length).each do |j|
+(2..num.length).each do |j| # O(n^2)
```

Comment:
A single loop can't be O(n^2), so this is actually O(n). And then you multiple the inner and outer loops to get O(n^2) overall.